lib/utils/debug-mode.ts

```diff
@@ -0,0 +1,7 @@
+export function isDebugModeEnabled() {
+	return (
+		process.env.DEBUG !== '0' &&
+		process.env.DEBUG !== '' &&
+		process.env.DEBUG !== undefined
```

Comment:
As for the logic itself, my two cents would be:
```
	return (
		!!process.env.DEBUG &&
		!['0', 'no', 'false'].includes(process.env.DEBUG.toLowerCase())
	);
```
I take note of your good reasoning in the linked issue:
> Now, i'm thinking supporting other "falsey" values only serves to add confusion. For example do we also support capitalization? export DEBUG=NO export DEBUG=YES?

I get it, but I think it doesn't align with "reducing friction" for users. We would not document / recommend anything other than DEBUG=1 (it would add confusion), but if someone out there in a hurry was to execute `export DEBUG=no` in a bash shell, your version of `isDebugModeEnabled()` would return `true`. Should it really? 

Re capitalization (good point!), `toLowerCase()` covers it.

Reply:
Yes, I see you point. One more alternative would be to stick to the letter of the docs and ONLY `DEBUG=1` enables debug mode. Presumably developers learn about the `DEBUG` option from reading the docs at which point they learn that the acceptable value is `1` not `true` or `yes`. Then any other value besides `1` would NOT enable debug mode.

Reply:
I have updated the PR to assign DEBUG to falsy for negative values.